packages/kuma-gui/src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -141,6 +129,18 @@ const click = (e: MouseEvent) => {
     }
   }
 }
+const $ref = ref<InstanceType<typeof KTableView> | null>(null)
+const rewrite = () => {
+  const $el = $ref.value?.$el
+  if (
+    ((el: any | HTMLElement): el is HTMLElement => el && typeof el.querySelectorAll === 'function')($el)
+  ) {
+    const $trs = $el.querySelectorAll('tr[tabindex="0"]')
+      ;['tabindex'].forEach(attr => Array.from($trs).forEach(item => item.removeAttribute(attr)))
+  }
```

Comment:
We remove the `tabindex=0` that KTableView adds to all of our rows, because hitting enter on those rows does absolutely nothing. I had a quick go at making this work (so hitting enter on a row would fire the "action link") but then I could find a way to make it work so that shift+enter would open in a new window. In the end I decided that if we can't replicate the native experience completely I'd rather not have keypress-able rows, you can still tab to the link itself and have all the native features work correctly (link shift+enter etc)